rust/bitcoin_wallet/src/bitcoin_wallet_assets/src/receive.html

```diff
@@ -0,0 +1,6 @@
+<h2>Receive Bitcoin</h2>
+<button id="backToDashboardBtn">Back to dashboard</button> 
+<section>
+  Address: <span id="address"></span> <button id="copyBtn">Copy</button>
+</section>
+<!-- TODO (ER-2527) Derive Bitcoin addresses for users (have to derive multiple addresses for a given principal) -->
```

Comment:
I'm not a fan of TODOs in example code. If you say it should stay, that's ok but I don't want it here by accident.